CMakePresets.json

```
@@ -0,0 +1,216 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 21,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "nix-base",
+      "hidden": true,
+      "generator": "Ninja",
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER_LAUNCHER": "ccache",
+        "CMAKE_FIND_FRAMEWORK": "LAST",
+        "CMAKE_FIND_PACKAGE_PREFER_CONFIG": "ON",
+        "CMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY": "OFF",
+        "CMAKE_FIND_USE_PACKAGE_REGISTRY": "OFF",
+        "CMAKE_OSX_SYSROOT": "",
+        "VAST_ENABLE_DEVELOPER_MODE": "ON",
+        "VAST_ENABLE_LSVAST": "ON",
+        "VAST_ENABLE_MDX_REGENERATE": "ON",
+        "VAST_PLUGINS": "${sourceDir}/plugins/broker;${sourceDir}/plugins/pcap;${sourceDir}/plugins/sigma;${sourceDir}/plugins/summarize"
+      }
+    },
+    {
+      "name": "clang-base",
+      "hidden": true,
+      "cacheVariables": {
+        "CMAKE_C_COMPILER": "clang",
+        "CMAKE_CXX_COMPILER": "clang++"
+      },
+      "environment": {
+        "LDFLAGS": "-fuse-ld=lld"
+      }
+    },
+    {
+      "name": "nix-clang-debug",
+      "inherits": ["nix-base", "clang-base"],
+      "displayName": "Clang Debug Config",
```

```suggestion
      "displayName": "Nix Clang Debug Config",
```

Since it inherits from the nix-base it should also specify that it's for Nix in the display name. The same applies to all the other ones below.